pillar/params.sls

```diff
@@ -158,6 +158,9 @@ cloud:
     lb_mon_retries: '3'
     # OpenStack Cinder Block Storage API version
     bs_version:     'v2'
+    # Ignore OpenStack Cinder avability zone when attaching volumes. 
+    # When Nova and Cinder have different availability zones, this should be set to true.
+    ignore_vol_az:  'false'
```

Comment:
Is there a reason for this to be a string instead of a boolean?

Reply:
I would prefer to keep it as it is, we use this same format for other boolean variables.

Reply:
agree, the `addons` in this file also have the same scheme